src/v2/guide/components-slots.md

```diff
@@ -235,7 +235,7 @@ Vue 实现了一套内容分发的 API，这套 API 的设计灵感源自 [Web C
 </span>
 ```
 
-我们想让它的后备内容显示用户的名，以取代正常情况下用户的姓，如下：
+我们想让它的后备内容显示用户的姓，以取代正常情况下用户的名，如下：
```

Comment:
> We might want to replace this fallback content to display the user's first name, instead of last, like this:

原文意思是：我们可能想换掉备用内容，用名而非姓来显示。

原来的翻译和修改后的都不对。